package.json

```diff
@@ -60,7 +61,7 @@
     "test-with-coverage": "nyc grunt test",
     "upload-coverage": "cat ./coverage/lcov.info | codacy-coverage",
     "clean-coverage": "rm -rf ./coverage .nyc_cache .nyc_output",
-    "postinstall": "bash copy-vendor.sh"
```

Comment:
We should remove this old `sh` file then since makes no sense to maintain multiple versions

Reply:
Thanks, I noticed it too, but forgot to mention.

Reply:
Done, just waiting on first question to push.